src/modules/News/validation.ts

```diff
@@ -10,12 +10,30 @@ const latestNews = {
   }),
 };
 
+const personalizedNews = {
+  query: Joi.object({
+    categories: Joi.string().required(),
+    limit: Joi.number().integer().min(1).max(100).default(10),
+    page: Joi.number().integer().min(1).max(100).default(1),
+  }),
+};
+
 const headlines = {
   query: Joi.object({
     country: Joi.string().optional().length(2),
-    category: Joi.string().optional(),
+    category: Joi.string()
+      .valid(
+        'business',
+        'entertainment',
+        'general',
+        'health',
+        'science',
+        'sports',
+        'technology',
+      )
+      .required(),
```

Comment:
@bojana-petroska you added the categories validation to the `headlines `which is great, but you have to also add it to `personalizedNews`, you could create an array and add these values to it, and then destrcture it `.valid(...arrayName)` to avoid repeating categories list.